pdaltools/add_points_in_pointcloud.py

```diff
@@ -35,37 +37,80 @@ def parse_args(argv=None):
         default=1000,
         help="width of tiles in meters",
     )
+    parser.add_argument(
+        "--spacing",
+        type=float,
+        default=0,
+        help="spacing between generated points in meters",
+    )
+    parser.add_argument(
+        "--altitude_column",
+        "-z",
+        type=str,
+        required=True,
+        default="RecupZ",
+        help="altitude column name from input geometry",
+    )
```

Comment:
Pour vraiment rendre `altitude_column` optionnel il faudrait le faire ici : 
```suggestion
    parser.add_argument(
        "--altitude_column",
        "-z",
        type=str,
        required=False,
        default=None,
        help="altitude column name from input geometry",
    )
```